ibctest/test_setup.go

```diff
@@ -271,13 +271,13 @@ func StartChainsAndRelayerFromFactory(
 
 func WaitForBlocks(srcChain, dstChain ibc.Chain, blocksToWait int64) error {
 	chainsConsecutiveBlocksWaitGroup := errgroup.Group{}
-	chainsConsecutiveBlocksWaitGroup.Go(func() (err error) {
```

Comment:
I personally always avoid naked returns. Why? https://levelup.gitconnected.com/go-naked-returns-4e2094b598e6

In past projects, I typically run a linter on CI that prohibts naked returns entirely.

In this case, I'm being pedantic.